src/test/java/org/sonar/plugins/checkstyle/metadata/CheckstyleMetadataTest.java

```diff
@@ -142,12 +139,15 @@ private static String convertName(String name) {
 
     private static String convertInternalKey(ModuleDetails moduleDetails) {
         String result = "Checker/";
-        if ("Checker".equals(moduleDetails.getParent())) {
+        if (moduleDetails.getParent().contains("Checker")) {
```

Comment:
keep `equals`

Reply:
Done

Reply:
please recheck that you pushed changes, I still see `contains`

Reply:
Done in new reorganized PR https://github.com/checkstyle/sonar-checkstyle/pull/306